Makefile

```diff
@@ -282,7 +282,7 @@ build-docker-localnode:
 # Run a 4-node testnet locally
 localnet-start: localnet-stop
 	@if ! [ -f build/node0/config/genesis.json ]; then docker run --rm -v $(CURDIR)/build:/tendermint:Z tendermint/localnode testnet --v 4 --o . --populate-persistent-peers --starting-ip-address 192.167.10.2 ; fi
-	docker-compose up
+	docker-compose up -d
```

Comment:
I'd like to leave this attached to `STDOUT` by default. If I am developing something and want to test Tendermint, I'd prefer to see the output instantly and not search some place for logs.